Modules/Core/SpatialObjects/include/itkSpatialObjectToImageFilter.h

```diff
@@ -188,6 +196,7 @@ class ITK_TEMPLATE_EXPORT SpatialObjectToImageFilter : public ImageSource<TOutpu
   void
   GenerateData() override;
 
+  IndexType     m_Index;
   SizeType      m_Size;
```

Comment:
That is a line that wasn't touching in this pull request, and there are many other member variables that would also need to be updated throughout this and many other classes to have a new standard of initializing member variables at declaration instead of in the constructor.   Right now it would be confusing to devs to have some variables initialized in the .h and others in the constructor - or potentially having constructor initializations being different than the .h.   So, for now, keeping as is.